src/sentry/integrations/slack/webhooks/action.py

```diff
@@ -382,6 +383,78 @@ def build_archive_modal_payload(self, callback_id):
             "callback_id": callback_id,
         }
 
+    def build_format_options(self, options: dict[str, str]) -> list[dict[str, dict[str, str]]]:
```

Comment:
The return type hint should be `list[dict[str, Any]]`, I think. (The dict at `"text"` has a `bool` as one of its values, and `"value"` maps to a `str` rather than a `dict`.)